openbb_platform/providers/polygon/openbb_polygon/models/stock_nbbo.py

```diff
@@ -146,13 +122,15 @@ class PolygonStockNBBOData(StockNBBOData):
         mode="before",
         check_fields=False,
     )
+    @classmethod
     def date_validate(cls, v):  # pylint: disable=E0213
         """Return formatted datetime."""
-        return (
-            to_datetime(v, unit="ns", origin="unix", utc=True).tz_convert("US/Eastern")
-            if v
-            else None
-        )
+
+        if v:
+            dlt = timedelta(hours=-5)
```

Comment:
@hjoaquim, can we do this with the TZ name, "America/New_York", instead of numbers?  If we rely on N hours, then this will be incorrect for half of the year, when daylight savings knocks off an hour.

Reply:
I agree. Was trying to get rid of the pandas import, but maybe that's the way!